packages/namekit-react/src/components/ENSTextarea.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { Textarea, TextareaProps } from "./Textarea";
+
+const ensInputClasses = "ens-webfont";
+
+export const ENSTextarea: React.FC<TextareaProps> = (props) => {
+  return (
+    <Textarea
+      {...props}
+      className={`${props.className || ""} ${ensInputClasses}`}
```

Comment:
Do we need to use `cc` in a situation like this? Appreciate your advice.

Reply:
Yeah I'll switch it out to use `cc`, thanks for pointing out.